Gulpfile.mjs

```diff
@@ -185,12 +190,14 @@ function buildRollup(packages) {
   );
 }
 
+const babelParserVersionNumber = await import("./packages/babel-parser/package")
```

Comment:
```suggestion
const babelParserVersionNumber = await import("./packages/babel-parser/package.json")
```
ESM require the complete file name

Btw, I'd prefer to move these two imports to the top-level and use normal import declarations (`import babelParserPkg from "./packages/babel-parser/package.json") rather than dynamic import.